test/push-encrypt/initial-state.json

```diff
@@ -0,0 +1,11 @@
+{
+  "allBranchesConfigs": [],
+  "branches": [
+    {
+      "branch": {
+        "name": "Main",
+        "isDefault": true
+      }
+    }
+  ]
+}
```

Comment:
z testu som musel vyhodit `ecpected-state.json` pretoze nevedel validovat zasifrovanu hodnotu cez wildcard `"#pass": "KBC::ProjectSecure%A::%s"`
https://github.com/keboola/keboola-as-code/pull/125/commits/87ae85cc6bf1d20720a3563f3ede1678765cb53d

Reply:
jj, ok, tam to nie je implementovane, .... ked bude niekedy cas, tak to mozem pridat.